seqr/models.py

```diff
@@ -169,8 +170,8 @@ class Project(ModelWithGUID):
 
     # user groups that allow Project permissions to be extended to other objects as long as
     # the user remains is in one of these groups.
-    can_edit_group = models.ForeignKey(Group, related_name='+', on_delete=models.PROTECT)
-    can_view_group = models.ForeignKey(Group, related_name='+', on_delete=models.PROTECT)
+    can_edit_group = models.ForeignKey(Group, related_name='+', on_delete=models.CASCADE, null=True, blank=True)
+    can_view_group = models.ForeignKey(Group, related_name='+', on_delete=models.CASCADE, null=True, blank=True)
```

Comment:
We overrode the delete function so every time a project was deleted its permission group was deleted, and its much cleaner to just let django handle that instead of us needing to maintain custom logic for it. A project permission group with no project is useless. It would always have been cleaner to do it this way, but since we are making this nullable we would have had to edit the custom to delete function to check for nulls so it would have gotten even messier

Reply:
oh oops. I got it backwards. Good catch!